src/shogun/preprocessor/RandomFourierGaussPreproc.cpp

```diff
@@ -14,120 +14,103 @@
 
 using namespace shogun;
 
+//TODO: Haven't handled this 
 void RandomFourierGaussPreproc::copy(const RandomFourierGaussPreproc & feats) {
```

Comment:
what about removing this method?
We can either have a copy ctor, or (and I think this is sufficient) just rely on clone